src/pykeen/training/slcwa.py

```diff
@@ -83,17 +89,16 @@ def _process_batch(
         if slice_size is not None:
             raise AttributeError('Slicing is not possible for sLCWA training loops.')
 
-        # Send positive batch to device
-        positive_batch = batch[start:stop].to(device=self.device)
-
-        # Create negative samples, shape: (batch_size, num_neg_per_pos, 3)
-        negative_batch, positive_filter = self.negative_sampler.sample(positive_batch=positive_batch)
+        # split batch
+        positive_batch, negative_batch, positive_filter = batch
```

Comment:
regardless of whether we use collation in the data loader, wouldn't it be better to update the interface to look like this?